src/pages/index.js

```diff
@@ -68,9 +68,9 @@ function Feature({ imageUrl, url, title, description }) {
     <div className={clsx("col col--4 component-block", styles.feature)}>
       {imgUrl && (
         <div className="text--center">
-          <a href={url}>
+          <Link to={useBaseUrl(url)}>
```

Comment:
Two changes here, both of them will be done in main instead of the archive branches: 

1. use `Link` instead of `a`, to take advantage of client-side routing. The transition looks much snappier.
2. `useBaseUrl` to wrap the `url`, so that the `1.3/` gets applied to the front of the path. (In a non-archived branch, this would still result in a valid URL, because the baseURL in main is `/`.)